scripts/validate_string_concatenation.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+"""
+Check where there is a string that needs to be concatenated.
+"""
+
+import os
+import sys
+import token
+import tokenize
+
+FILE_EXTENTIONS_TO_CHECK = [".py", ".pyx"]
```

Comment:
What other file types should be included?

Reply:
Maybe .pxd and and .pxi.ini?